app/domain/logs.rb

```diff
@@ -670,27 +671,27 @@ module AuthnJwt
       )
 
       ConvertingClaimAccordingToMapping = ::Util::TrackableLogMessageClass.new(
-        msg: "Converting claim '{0-claim-name}' to '{1-annotation-name}'...",
+        msg: "Converting claim '{0-claim-name}' to annotation '{1-annotation-name}'...",
         code: "CONJ00133D"
       )
 
       EnforcedClaimsToBeChecked = ::Util::TrackableLogMessageClass.new(
-        msg: "Enforced claims to be checking in host are '{0-enforced-claims}'",
+        msg:  "Enforced claims to be checking in host are '{0-enforced-claims}'",
         code: "CONJ00134I"
       )
 
       ValidatedEnforcedClaimsConfiguration = ::Util::TrackableLogMessageClass.new(
-        msg: "Successfully validated enforced claims configuration",
+        msg:  "Successfully validated enforced claims configuration",
```

Comment:
fine. Should there be that extra space before  "Successfully..."

Reply:
Nop. IIt's for marking for review :-)